src/modules/shortcut_guide/overlay_window.cpp

```diff
@@ -505,15 +510,24 @@ void D2DOverlayWindow::hide_thumbnail() {
 }
 
 void D2DOverlayWindow::render(ID2D1DeviceContext5* d2d_dc) {
+  auto current_anim_value = animation.value(Animation::AnimFunctions::LINEAR);
   if (!winkey_held() || is_start_visible()) {
-    hide();
-    instance->was_hidden();
-    return;
+    if (hiding) {
+      if (animation.done()) {
+        hide();
+        instance->was_hidden();
+        animation.reset(animation_time, 0.0, 1.0);
+        hiding = false;
+        return;
+      }
+    } else {
+      animation.reset(animation_time / 2.0, current_anim_value, 0.0);
+      hiding = true;
```

Comment:
Shouldn't `current_anim_value` be reset here?

Reply:
I'm not sure why?